dev/mkmrconfig.sh

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env bash
+set -eo pipefail; [[ $MKMRCONFIG_TRACE ]] && set -x
+
+# Scans Github organizations and emits a config file for use with `myrepos`.
+#
+# Dependencies:
+#   myrepos -- https://myrepos.branchable.com
```

Comment:
This looks like it isn't updated frequently. Can you find the github repo and point to it?